sw/device/lib/dif/dif_rv_core_ibex.c

```diff
@@ -205,3 +205,32 @@ dif_result_t dif_rv_core_ibex_clear_error_status(
 
   return kDifOk;
 }
+
+dif_result_t dif_rv_core_ibex_get_rng_status(
```

Comment:
to be consistent with the name of the `enum` "rng" -> "rnd" & object_verb style of naming:

```suggestion
dif_result_t dif_rv_core_ibex_rnd_status_get(
```